### ISSUE TITLE
Using autoloader

### ISSUE THREAD

**Issue by ferishili** [opencast-ilias/OpencastEvent#10]
**Is your feature request related to a problem? Please describe.**
Currently OpencastEvent does not have any autoloader and classes and namespaces are somehow isolated. The functionality and accessibility is also very limited, and this needs to be solved by introducing composer `autoloader`

**Describe the solution you'd like**
All classes must get the relative namespaces and the inclusions and requires must be switched to using `classmaps` etc.

**Additional context**
This has also the advantage of increasing performance as well as providing a vast features and usability to/from the plugin.
